docker-fedora/README.md

```diff
@@ -0,0 +1,53 @@
+# container-engine
+
+This is the container engine container intended to be used as a system container for fedora systems.
+
+Note this only provides the *daemon process*, and does not include CLI tools.
+
+## Building the image from source:
+
+```
+# git clone https://github.com/projectatomic/atomic-system-containers
+# cd atomic-system-containers/docker-fedora
+# docker build -t container-engine .
+```
+
+## Running the system container, with the atomic CLI:
+
+Requirement: etcd and flannel services are running
```

Comment:
I don't think that these are needed for the image to work. They would be needed for origin/openshift/etc.. though.